src/main/java/ratpack/zipkin/ServerTracingModule.java

```diff
@@ -55,11 +55,28 @@ protected void configure() {
         .to(DefaultServerTracingHandler.class)
         .in(Singleton.class);
 
-    bind(HttpClient.class).annotatedWith(Zipkin.class)
-        .to(ZipkinHttpClientImpl.class)
-        .in(Singleton.class);
+    bind(ClientTracingInterceptor.class)
+            .to(DefaultClientTracingInterceptor.class)
+            .in(Singleton.class);
+
+    Provider<ClientTracingInterceptor> clientTracingInterceptorProvider =
+            getProvider(ClientTracingInterceptor.class);
+
+    Provider<HttpClient> httpClientProvider = () ->
+            // HttpClient.of throws exceptions but Providers can not, hence the unchecked wrapper.
+            Exceptions.uncheck(() -> HttpClient.of(spec -> {
+              spec.requestIntercept(clientTracingInterceptorProvider.get()::request);
+              spec.responseIntercept(clientTracingInterceptorProvider.get()::response);
+            }));
 
-    bind(ZipkinHttpClientImpl.class);
+    bind(HttpClient.class)
```

Comment:
This changes behavior in that all bindings on HttpClient will now use one that is configured for tracing.  This may be a confusing change if folks are use to the required `@Zipkin` annotation.  In practice however, I have found it more likely that folks forget the `@Zipkin` annotation and wonder why there is no client tracing.